doc/developer/design/20230126_designs_reviews.md

```diff
@@ -0,0 +1,91 @@
+Engineers write designs in Notion because it's easy to start, but we lack any established review process, and following discussions is difficult.
+Writing designs as pull-requests is perceived to be more tedious than simply writing the content in Notion.
+To address this, we analyze the current shortcomings and propose updates to the existing process.
+
+# Explanation
+[explanation]: #explanation
+
+We aim at creating a design process that is lightweight, captures reviews and specific snapshots in time, is open, while providing a clear benefit to engineers.
+Let's break down these aspects.
+
+A design review process needs to empower engineers to focus on the task at hand.
+It helps to talk about goals, how they fit into the product roadmap, and outline solutions.
+An engineer can present different solutions, and it's expected that new alternatives come up while working on a design.
+Explaining trade-offs helps to motivate a specific solution.
+
+Creating a design is an iterative process.
+Capturing discussions is an important aspect to understand how an opinion was formed and how designs changed during the review.
+We need to capture both, and Notion doesn't give us a good way to capture discussions and specific versions.
+
+Materialize defaults to leading discussions in the open, unless it's not permissible.
+This ensures that we uphold a high standard, and it increases the visibility of each engineer's contributions.
+Designs in Notion do not achieve this goal.
+
+The most important aspect is that this process provides a benefit to the engineer and the company.
+It allows us to create confidence that their work fits into the overall architecture and that dependencies are surfaced.
+Additionally, a good design document serves as a point-in-time documentation, helping others understand trade-offs and seemingly odd choices.
+
+# Reference explanation
+[reference-explanation]: #reference-explanation
+
+The current design process fulfills most of the requirements outlined above.
+It is lightweight, encourages open discussions and has short review cycles.
+This raises the question of why we did not follow it recently.
+
+To address this, we propose the following steps, some of which are part of the change around this document:
+* Revisit the design doc [README](./README.md) to make expectations clearer.
+* Update the [template](./00000000_template.md) to match what we're outlining in this design.
+* Update the [pull request template](/.github/pull_request_template.md) to include a step reminding engineers to write a design document.
+
+Specifically, the current interpretation of design documents is to write one for large changes, where it is not clear what large means.
+Instead, we propose that each change should come with a design doc unless it is small enough to be non-contentious or immediately clear what needs to be done.
+By default, engineers should write design docs.
```

Comment:
Maybe give a hand-wavey sizing here? E.g. if you think your work will take more than 'x' weeks or has potential tradeoffs you probably want a design doc

Reply:
ah, I see it referenced in the design readme